pressbooks/pressbooks-book
# Improve 'About the Author' section

This PR changes the HTML structure and Buckram SCSS defaults for the new 'About the Author' section which displays contributor information at the end of webbook chapters when the theme option has been selected.

Fix for: https://github.com/pressbooks/pressbooks-book/issues/854

**To Test:**
1. Checkout https://github.com/pressbooks/pressbooks/tree/pb-2371-about-author-export until https://github.com/pressbooks/pressbooks/pull/2372 has been merged to dev?
2. Check out this branch and regenerate your theme/change to another theme based on Buckram
3. Create multiple contributors and add desired contributor info (profile picture, institution, links, bio, etc.)
4. Assign one or more of these contributors as authors of specific book content (front/back matter, chapter)
5. Turn on the 'About the Author' global theme setting for your book
6. Visit content which has authors assigned and inspect the new 'About the Author(s)' section displayed at the bottom of the chapter
7. Change to a new theme and do step 6 again
